connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/processors/bop/BOPRequestPreparer.java

```diff
@@ -66,25 +65,26 @@ public void process(final Exchange exchange) {
             Log.infof("Recipients: %s", Arrays.toString(recipients.toArray()));
         }
 
+        // Prepare the email to be sent.
         final Email email = new Email(
             subject,
             body,
             recipients
         );
 
-        JsonObject bopBody;
-        if (emailConnectorConfig.isSkipBopUsersResolution()) {
-            final SendEmailsRequest request = new SendEmailsRequest();
-            request.addEmail(email);
-            bopBody = JsonObject.mapFrom(request);
-        } else {
-            final Emails emails = new Emails();
```

Comment:
Let's delete the `Emails` class if it's no longer used in the connector.

Reply:
Done!